### ISSUE TITLE
end-of-life to unmaintained?

### ISSUE THREAD

**Issue by k-okada** [ros-gbp/bfl-release#12]
hi,
can I take over maintainer role of bfl as  ros-orphaned-packages@googlegroups.com ? if you give me access to this package, i'll change maintainer and status tag, and check if https://github.com/ros-infrastructure/ros_buildfarm/pull/435 works.

**Comment by tfoote** [comment]
I've added you as a collaborator.

**Comment by k-okada** [comment]
Thank you, it seems you have already released into lunar, so we'll released
from M

--
◉ Kei Okada


2017-05-17 8:33 GMT+09:00 Tully Foote <notifications@github.com>:

**Comment by tfoote** [comment]
From the discussion in #11 it likely won't need release into M turtle. It's only used by one other EOL package. It would be way better for the community to focus on phasing out robot_pose_ekf than rereleasing this package.

**Comment by jacknlliu** [comment]
It sounds a little sad. How many packages will leave with ROS in the future to powerful the world wide robots should be worthy of consideration.

**Comment by k-okada** [comment]
@jacknlliu if you plan to use this package in future, please let me know, we're happy to release in future ROS-distro

**Comment by dlaz** [comment]
[people_tracking_filter](https://github.com/wg-perception/people/blob/indigo-devel/people_tracking_filter/package.xml) also depends on bfl. If someone could do a release, that would be great, as I'd like to people_tracking_filter (and all of people) for melodic.

Is there something blocking bfl from getting into ubuntu main? I see debs are already being built: https://people.mech.kuleuven.be/~tdelaet/bfl_doc/installation_guide/node2.html

@DLu fyi

**Comment by k-okada** [comment]
@dlaz i think it is already released in indigo, kinetic and lunar https://github.com/ros-gbp/bfl-release

**Comment by dlaz** [comment]
Yes, but not melodic, which is what I'm interested in, or am I misunderstanding something?

**Comment by k-okada** [comment]
ok, done https://github.com/ros/rosdistro/pull/17774


--
◉ Kei Okada


2018-05-08 1:36 GMT+09:00 Dan Lazewatsky <notifications@github.com>:

**Comment by dlaz** [comment]
Awesome, thank you.

**Comment by sloretz** [comment]
Since `liborocos-bfl-dev` is availabile in the `Noetic` target platforms Debian Buster and Ubuntu Focal, can this package be replaced with a rosdep key in `Noetic`?

* https://packages.debian.org/buster/liborocos-bfl-dev
* https://packages.ubuntu.com/focal/liborocos-bfl-dev

**Comment by k-okada** [comment]
@sloretz that's would be great. Did someone check if dependency package such as (http://wiki.ros.org/robot_pose_ekf?distro=melodic) works with `liborocos-bfl-dev` ?

**Comment by sloretz** [comment]
> Did someone check if dependency package such as (http://wiki.ros.org/robot_pose_ekf?distro=melodic) works with liborocos-bfl-dev ?

@k-okada I just checked; `robot_pose_ekf` tests pass on Ubuntu Focal and Debian Buster using `liborocos-bfl-dev`

**Comment by sloretz** [comment]
ROS Noetic will use rosdep keys for BFL and KDL. The **[ROS Noetic migration notes](http://wiki.ros.org/noetic/Migration#Use_Orocos_KDL_and_BFL_rosdep_keys)** has instructions.
